src/SFA.DAS.QnA.Api/Controllers/ApplicationSnapshotController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.QnA.Api.Infrastructure;
+using SFA.DAS.QnA.Application.Commands.CreateSnapshot;
+
+namespace SFA.DAS.QnA.Api.Controllers
+{
+    [Route("/applications")]
+    [Produces("application/json")]
+    [ApiController]
+    public class ApplicationSnapshotController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public ApplicationSnapshotController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        /// <summary>
+        ///     Creates a snapshot of the requested application. Used for RoATP
```

Comment:
Ugh, client specific comment. 😉

Reply:
yeah it's a big stinky stain within QnA 😿 I've added some remarks to further expand on why the endpoint is there